src/parser/openapi.parser.ts

```diff
@@ -82,7 +82,7 @@ export abstract class OpenApiParser {
 
     const components = document.components ? this.parseComponents(document.components) : {};
 
-    const paths = (document.paths ? this.parsePaths(document.paths, components) : []).sort((a, b) => a.name.localeCompare(b.name));
+    const paths = document.paths ? this.parsePaths(document.paths, components) : [];
```

Comment:
alternatively, we could remove the sort here and the one you added below so that it will use the order defined in the openapi spec.